### PR TITLE
R-lang : new recipe

### PR DESCRIPTION
**Pull request by miqlas**
R is a programming language and software environment for statistical computing and graphics supported by the R Foundation for Statistical Computing. The R language is widely used among statisticians and data miners for developing statistical software and data analysis. Polls, surveys of data miners, and studies of scholarly literature databases show that R's popularity has increased substantially in recent years.

R is a GNU package. The source code for the R software environment is written primarily in C, Fortran, and R. R is freely available under the GNU General Public License, and pre-compiled binary versions are provided for various operating systems. While R has a command line interface, there are several graphical front-ends available.

### CODE REVIEW

File: dev-lang/r/r-3.3.2.recipe [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	r$secondaryArchSuffix = $portVersion
+	cmd:r$secondaryArchSuffix
+	cmd:rscript$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	#lib:libcairo$secondaryArchSuffix
```
Comment (waddlesplash) [inline dev-lang/r/r-3.3.2.recipe]: why is this commented out?

File: dev-lang/r/r-3.3.2.recipe [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+	lib:libbz2$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	#devel:libblas$secondaryArchSuffix
+	#devel:liblapack$secondaryArchSuffix
+	#devel:libcairo$secondaryArchSuffix
```
Comment (waddlesplash) [inline dev-lang/r/r-3.3.2.recipe]: Same here.

File: dev-lang/r/r-3.3.2.recipe (hunk 1 above)
Comment (miqlas) [inline dev-lang/r/r-3.3.2.recipe]: Because our current cairo port just too old.

File: dev-lang/r/r-3.3.2.recipe (hunk 2 above)
Comment (waddlesplash) [inline dev-lang/r/r-3.3.2.recipe]: @miqlas ok, so why are the blas/lapack lines commented out here then?

Comment (miqlas) [inline dev-lang/r/r-3.3.2.recipe]: they aren't compiled with fortran and i don't want to mix them.
R comes with intree blas and lapack, so it have less dependecies.

Comment (waddlesplash) [inline dev-lang/r/r-3.3.2.recipe]: So why not remove them?

Comment (miqlas) [inline dev-lang/r/r-3.3.2.recipe]: Because i want to recompile them with fortran. It is a nice test for gfortran. But it takes time.

**Pull request merged**